### PR TITLE
chore(release): fix PR and Jira links in 3.8.0 changelog

### PR DESCRIPTION
**Pull request by ADD-SP**
### Summary

Fix https://github.com/Kong/kong/issues/13763

### Checklist

- [N/A] The Pull Request has tests
- [N/A] A changelog file has been created under `changelog/unreleased/kong` or `skip-changelog` label added on PR if changelog is unnecessary. [README.md](https://github.com/Kong/gateway-changelog/blob/main/README.md)
- [N/A] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Issue reference

_[KAG-5596]_

[KAG-5596]: https://konghq.atlassian.net/browse/KAG-5596?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,39 +30,42 @@ Individual unreleased changelog entries can be located at [changelog/unreleased]
 
 - Fixed an inefficiency issue in the Luajit hashing algorithm
  [#13240](https://github.com/Kong/kong/issues/13240)
- 
+ [KAG-4646](https://konghq.atlassian.net/browse/KAG-4646)
```
Comment (AndyZhang0707) [inline CHANGELOG.md:33]: We need to remove these internal tickets from this file.

Comment (ADD-SP) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -99,7 +105,8 @@ which would improve the latency significantly.
 
 
 - Bumped `Wasmtime` version to `23.0.2`
- [#12011](https://github.com/Kong/kong/issues/12011)
+ [#12930](https://github.com/Kong/kong/issues/12930)
```
Comment (AndyZhang0707) [inline CHANGELOG.md:108]: I checked the original PR, but neither of them bumped the version to `23.0.2`.

**Review by AndyZhang0707 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (ADD-SP) [reply CHANGELOG.md]: fixed

**Review by AndyZhang0707 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -61,7 +64,8 @@ Individual unreleased changelog entries can be located at [changelog/unreleased]
  
 
 - Bumped lua-resty-healthcheck from 3.0.1 to 3.1.0 to fix an issue that was causing high memory usage
- [#13532](https://github.com/Kong/kong/issues/13532)
+ [#13038](https://github.com/Kong/kong/issues/13038)
+ [FTI-5847](https://konghq.atlassian.net/browse/FTI-5847)
```
Comment (ADD-SP) [inline CHANGELOG.md:68]:
```suggestion
```

**Review by AndyZhang0707 — APPROVED** [review]

**Review by ProBrian — APPROVED** [review]

**Pull request merged**